third_party/3/pyspark/sql/functions.pyi

```diff
@@ -97,8 +97,8 @@ def from_unixtime(timestamp: ColumnOrName, format: str = ...) -> Column: ...
 def unix_timestamp(
     timestamp: Optional[ColumnOrName] = ..., format: str = ...
 ) -> Column: ...
-def from_utc_timestamp(timestamp: ColumnOrName, tz: str) -> Column: ...
-def to_utc_timestamp(timestamp: ColumnOrName, tz: str) -> Column: ...
+def from_utc_timestamp(timestamp: ColumnOrName, tz: Union[Column, str]) -> Column: ...
```

Comment:
In general we have an alias `ColumnOrName`:

https://github.com/zero323/pyspark-stubs/blob/8cc075b11ffe887586d98a7c21a2708e559d8dfd/third_party/3/pyspark/sql/_typing.pyi#L27

which can be used in such cases. Could you use it here and below?

Reply:
thanks, updated!